docs/conceptual/runtime/srml.md

```diff
@@ -19,7 +19,7 @@ functionality for a runtime.
 
 ### System Module
 
-The [System module](https://substrate.dev/rustdocs/master/palette_system/index.html) provides low-level types, storage, and
```

Comment:
Lots to still be done in this file:
* Rename the entire file
* Rename all the headers eg System Module -> System Pallet
* rename srml -> ?? (brought this up on riot chat)

Reply:
Updated the title, this is meant only to change links.

Other steps you can help with (which include what you mention here) are listed here: https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/issues/324